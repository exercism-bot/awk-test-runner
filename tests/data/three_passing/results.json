{
  "version": 2,
  "status": "pass",
  "test-environment": {
    "gawk": "GNU Awk 5.1.0, API: 3.0 (GNU MPFR 4.1.0, GNU MP 6.2.1)",
    "bats": "Bats 1.7.0",
    "OS": "Ubuntu 22.04 LTS"
  },
  "tests": [
    {
      "name": "say one",
      "status": "pass",
      "test_code": "run bash three_passing.sh one\n[ \"$status\" -eq 0 ]\n[ \"$output\" == \"one\" ]"
    },
    {
      "name": "say two",
      "status": "pass",
      "test_code": "run bash three_passing.sh two\n[ \"$status\" -eq 0 ]\n[ \"$output\" == \"two\" ]"
    },
    {
      "name": "say three",
      "status": "pass",
      "test_code": "run bash three_passing.sh three\n[ \"$status\" -eq 0 ]\n[ \"$output\" == \"three\" ]"
    }
  ]
}
