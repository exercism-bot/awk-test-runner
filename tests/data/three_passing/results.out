1..3
ok 1 say one
ok 2 say two
ok 3 say three
